packages/clay-panel/src/index.tsx

```diff
@@ -49,9 +54,10 @@ export interface IProps extends React.HTMLAttributes<HTMLDivElement> {
 	displayTitle?: React.ReactNode;
 
 	/**
-	 * Flag to indicate the visual variation of the Panel.
+	 * Display type. Deprecation warning: previous "unstyled" and "secondary"
+	 * styles are not longer supported.
```

Comment:
Wordsmithing: "previous "unstyled" and "secondary" styles are not longer supported." --> ""unstyled" and "secondary" variants are deprecated. They have been renamed to "default" and "block"."

Reply:
Done!